content/body/screen-reader-only-text.php

```diff
@@ -75,8 +75,8 @@ class="enable-example"
 
 <ol>
   <li><strong>When giving an accessible name or label to a group of elements (using the <code>group</code> role) or a landmark</strong>. A good example is in step #4 of <a href="/multi-level-hamburger-menu.php#developer-walkthrough-2">the code walkthrough for the explanation of what makes the Enable Hamburger Menu accessible</a>.</li>
-  <li><strong>To update the semantic meaning of an element coded with an ARIA <code>role</code> attribute</strong>, such as a step #1 of <a href="progress.php#developer-walkthrough-3">Enable ARIA progress bar example</a>.</li>
-  <li><strong>When an accessible label is missing from a control.</strong>  For example, if you have a button that just a background image or an icon font (you could use screen reader only text for this too, but all modern screen readers can understand ARIA labels on interactive elements)</li>
+  <li><strong>To update the semantic meaning of an element coded with an ARIA <code>role</code> attribute</strong>, such as step #1 of <a href="progress.php#developer-walkthrough-3">Enable ARIA progress bar example</a>.</li>
+  <li><strong>When an accessible label is missing from a control.</strong>  For example, if you have a button that is just a background image or an icon font (you could use screen reader only text for this too, but all modern screen readers can understand ARIA labels on interactive elements)</li>
```

Comment:
Can we add a period at the end of this sentence?

```suggestion
  <li><strong>When an accessible label is missing from a control.</strong>  For example, if you have a button that is just a background image or an icon font (you could use screen reader only text for this too, but all modern screen readers can understand ARIA labels on interactive elements).</li>
```